core/capabilities/ccip/ccipsolana/commitcodec.go

```diff
@@ -110,16 +110,21 @@ func (c *CommitPluginCodecV1) Decode(ctx context.Context, bytes []byte) (cciptyp
 		return cciptypes.CommitPluginReport{}, err
 	}
 
-	merkleRoots := []cciptypes.MerkleRootChain{
-		{
-			ChainSel:      cciptypes.ChainSelector(commitReport.MerkleRoot.SourceChainSelector),
-			OnRampAddress: commitReport.MerkleRoot.OnRampAddress,
-			SeqNumsRange: cciptypes.NewSeqNumRange(
-				cciptypes.SeqNum(commitReport.MerkleRoot.MinSeqNr),
-				cciptypes.SeqNum(commitReport.MerkleRoot.MaxSeqNr),
-			),
-			MerkleRoot: commitReport.MerkleRoot.MerkleRoot,
-		},
+	var merkleRoots []cciptypes.MerkleRootChain
+	if commitReport.MerkleRoot == nil {
+		merkleRoots = nil
+	} else {
```

Comment:
Isn't this a redundant set because the default slice value is nil? Couldn't we just do 
```suggestion
	var merkleRoots []cciptypes.MerkleRootChain
	if commitReport.MerkleRoot != nil {
```